data-collection/src/main/java/refactoringml/ProcessMetricTracker.java

```diff
@@ -74,6 +77,29 @@ public void resetCounter(CommitMetaData commitMetaData) {
 
 	public ProcessMetrics getCurrentProcessMetrics() { return currentProcessMetrics; }
 
+	public int getCommitCountThreshold() { return currentCommitThreshold; }
+
+	//Filter class files that were not refactored in the last K commits and not already found with a lower K.
+	//TODO: If a class has an inner class only one instance is stored in the database
+	// The fix (>= instead of > : currentCommitThreshold) leads to stable commit duplicates in the DB in case of multiple refactorings in the current commit
+	public boolean calculateStability(List<Integer> commitThresholds){
```

Comment:
The method is called `calculateStability` but returns a boolean. Maybe a small doc here indicating what true and false means?

Reply:
Maybe we should separate the method that returns whether this is stable from the one that update the currentCommitThreshold.

[Mixing "command and query" in a single method is usually not a good idea](https://en.wikipedia.org/wiki/Command–query_separation).